packages/block-editor/src/components/link-control/style.scss

```diff
@@ -3,6 +3,12 @@
 	min-width: $modal-min-width;
 }
 
+// Provides positioning context for reset button. Without this then when an
+// error notice is displayed above the input the reset button is incorrectly positioned.
```

Comment:
I haven't tested the branch, but why would the notice be displayed above the input? I don't see how based on the rendering of the notice being after the input, and no associated styles, why it would be shown above.

Reply:
My bad. The comment is overly specific. The general intent is accurate, however.  I've amended to remove references to any specific layout (above/below...etc) and just noted that without this the positioning may be wrong.

See below which shows that errors can display when the input is visible:

<img width="393" alt="Screenshot 2020-02-05 at 13 16 23" src="https://user-images.githubusercontent.com/444434/73845071-bd90ff00-4819-11ea-9902-bbbd53df01b7.png">

You can test by:

* Opening Network tools and filtering by XHR
* Add new Link and create any random page.
* Notice the REST API request in the Network tools. Block this request (right click "Block this URL").
* Create another link and try create any random page.
* This time the REST API request gets blocked by Network tools and the error will show.
* Don't forget to unblock after testing!